llvm/tools/llvm-gsymutil/llvm-gsymutil.cpp

```diff
@@ -357,6 +357,11 @@ static llvm::Error handleObjectFile(ObjectFile &Obj, const std::string &OutFile,
   if (auto Err = DT.convert(ThreadCount, Out))
     return Err;
 
+  // Organize overlapping functions as children of top-level functions. Do this
+  // right after loading the DWARF data so we don't have to deal with functions
+  // from the symbol table.
```

Comment:
This comment makes it seam like we will merge any functions with overlapping ranges. Maybe some different wording here?:
```
// Merge functions with identical address ranges as merged functions in the 
// first FunctionInfo with that address range. Do this right after loading the 
// DWARF data so we don't have to deal with functions from the symbol table.
```